src/Sema.zig

```diff
@@ -17974,6 +17975,101 @@ fn zirWasmMemoryGrow(
         } },
     });
 }
+fn zirReflectDecl(
+    sema: *Sema,
+    block: *Block,
+    extended: Zir.Inst.Extended.InstData,
+) CompileError!Air.Inst.Ref {
+    const extra = sema.code.extraData(Zir.Inst.BinNode, extended.operand).data;
+
+    const src = LazySrcLoc.nodeOffset(extra.node);
+    const ty_src: LazySrcLoc = .{ .node_offset_builtin_call_arg0 = extra.node };
+    const name_src: LazySrcLoc = .{ .node_offset_builtin_call_arg1 = extra.node };
+
+    const namespace_ty = try sema.resolveType(block, ty_src, extra.lhs);
+    const decl_name = try sema.resolveConstString(block, name_src, extra.rhs);
+
+    const target = sema.mod.getTarget();
+
+    try checkNamespaceType(sema, block, ty_src, namespace_ty);
+
+    // Type must be a namespace because we just checked.
+    const namespace = namespace_ty.getNamespace().?;
+
+    if (try sema.lookupInNamespace(block, name_src, namespace, decl_name, true)) |decl_index| {
```

Comment:
```suggestion
    if (try sema.namespaceLookup(block, name_src, namespace, decl_name)) |decl_index| {
```

Is there a specific reason why you copied the check for `is_pub`?

Reply:
Be sure to delete the now unnecessary `is_pub` check.